packages/headless/src/composables/comboBox/createComboBox.ts

```diff
@@ -112,7 +138,7 @@ export const createComboBox = createBuilder(
           "aria-expanded": isExpanded.value,
           "aria-controls": controlsId,
           "aria-labelledby": labelId,
-          "aria-activedescendant": activeKey.value ? getOptionId(activeKey.value) : undefined,
+          "aria-activedescendant": activeOption.value ? getOptionId(activeOption.value) : undefined,
```

Comment:
Isn't this already set by the listbox?

Reply:
Not, if it is controlled. Please check the implementation.